src/marketing/links/link.scss

```
@@ -0,0 +1,60 @@
+.mktg-link {
```

We usually add `mktg` at the very end 👀 I think it makes sense to align with that convention here (unless we have a good reason not to!)

```suggestion
.link-mktg {
```

---

That makes sense! What would be the preference for the expanded class names, e.g. `.link-mktg-inner` or `link-inner-mktg`?

I would go with `mktg-link` so it can be a prefix you can search on, to find all the related elements and styles.